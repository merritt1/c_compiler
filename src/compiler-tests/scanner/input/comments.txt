/* this is a comment */
int a = 123;
/* this is a *** weirder comment let's see if it works ****/
"failsafe"
/* how about this one?/////*/
