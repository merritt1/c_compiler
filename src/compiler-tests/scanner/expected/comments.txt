loc = 0002:0001-0002:0003     text = int                      token = [INT                 ]
loc = 0002:0005-0002:0005     text = a                        token = [IDENTIFIER          ]     name = a
loc = 0002:0007-0002:0007     text = =                        token = [EQUAL               ]
loc = 0002:0009-0002:0011     text = 123                      token = [NUMBER              ]     type =   signed  int     value = [123       ]
loc = 0002:0012-0002:0012     text = ;                        token = [SEMICOLON           ]
loc = 0004:0001-0004:0010     text = "failsafe"               token = [STRING              ]     val = failsafe
